WORKSPACE

```diff
@@ -101,10 +101,10 @@ load("//defs:repo_rules.bzl", "fetch_kube_release")
 fetch_kube_release(
     name = "io_k8s_release",
     archives = {
-        "kubernetes-node-linux-amd64.tar.gz": "1870c8ec9a38af4d2e86d2ed22c3a4840e1bfe12aa08e7c70c9a86d21f04c375",
-        "kubernetes-manifests.tar.gz": "4bdcbde160458bb664bf682aad25f37a19cd1a2b2baa5a0f7bab216a11513d7d",
-        "kubernetes-server-linux-amd64.tar.gz": "aced340e01320ee9ae488bf7ad3ccb4853185316aa290597ca195754146d08af",
-        "kubernetes-node-windows-amd64.tar.gz": "c9b8e681ff5b2633e6ac0cfe66b1d0795b22b6daf51ec029f5793009c82fcd8c",
+        "kubernetes-node-linux-amd64.tar.gz": "c8d7ba8b68fbb83d9c0f9c97b30e7601ef37abb2bfa94ea699d28e55f844ffd7",
```

Comment:
How did you arrive at these values? Did you update both `tools/sha256_generator.sh` and `version.sh`?

When I was trying to do the bump myself, I continually got this error, though I admit I know next to nothing about how Bazel works.


```
❯ bazel clean && bazel build //release:release-tars
INFO: Starting clean.
ERROR: /Users/nbrubake/projects/cloud-provider-gcp/release/BUILD:115:18: configurable attribute "srcs" in //release:release-tars doesn't match this configuration. Would a default condition help?
```

Reply:
it may be updated but steps where documetned here https://github.com/kubernetes/cloud-provider-gcp/blob/master/docs/instruction-how-to-bump-repository.md cc: @mmamczur